pkg/apis/extensions/types.go

```diff
@@ -292,6 +292,48 @@ type DeploymentList struct {
 	Items []Deployment `json:"items"`
 }
 
+type DaemonSetUpdateStrategy struct {
+	// Type of daemon set update. Only "RollingUpdate" is supported at this time. Default is RollingUpdate.
+	Type DaemonSetUpdateStrategyType `json:"type,omitempty"`
```

Comment:
Yes, but we havent been able to get to it https://github.com/kubernetes/kubernetes/issues/3933